README.md

```diff
@@ -28,7 +28,6 @@ This repository contains the source for the following packages:
 - **[`@ni/nimble-tokens`](/packages/nimble-tokens/)** - design tokens used by the component packages
 
 And some additional utility packages:
```

Comment:
Looks like we need to fix `npm audit` again. Let me know if you want help with that.

Reply:
Passing now.